### ISSUE TITLE
File name

### ISSUE THREAD

**Issue by rodionstepanov** [erikrause/RSCIExportOJSPlugin#1]
Добрый день!

Было бы удобно иметь то имя файла статьи, которое указано в Гранках. 
П.С. большое спасибо за плагин. 
Удачи

**Comment by erikrause** [comment]
Спасибо за отзыв!

Изменил наименование файлов в архиве на имена из гранок: https://github.com/erikrause/RSCIExportOJSPlugin/releases/tag/0.1.0.1

**Comment by rodionstepanov** [comment]
Эта версия не встает. Говорит, что уже стоит более свежая версия плагина. У меня установлена OJS 3.3.0.10.

**Comment by rodionstepanov** [comment]
Извините за наглость, но было бы здорово еще экспортировать дату поступления (date_sabmitted) и финансирование (agencies).

**Comment by erikrause** [comment]
Да, тот архив работает на OJS 3.1.2. Вот версия для OJS 3.3:
[rsciexport.tar.gz](https://github.com/erikrause/RSCIExportOJSPlugin/files/8740770/rsciexport.tar.gz)

Я сейчас поддерживаю журнал на OJS 3.1.2, поэтому у меня нет возможности протестировать на OJS 3.3. Пожалуйста, проверьте работоспособность этой версии и сообщите, все ли в порядке.

По поводу agencies - в OJS 3.1 нет атрибута для указания источника финансирования у статей. В OJS 3.3 есть такая возможность? Или это через сторонний плагин? date_sabmitted должно быть легко сделать, но у меня нет сейчас времени этим заняться. Возможно, сделаю позже.

**Comment by rodionstepanov** [comment]
Нет, не работает. Страница не грузится при нажатии "Экспортировать ...".

Да,  в OJS 3.3 есть setting_name  submissionAgency
